app/routes/resources+/login.tsx

```diff
@@ -20,20 +20,22 @@ export const LoginFormSchema = z.object({
 	username: usernameSchema,
 	password: passwordSchema,
 	redirectTo: z.string().optional(),
-	remember: z.boolean(),
+	remember: z.boolean().default(false),
```

Comment:
Conform cannot resolve it to boolean automatically. If the remember checkbox is not checked, the preprocess would cast it to `false`.

```suggestion
	remember: z.preprocess(value => value === 'on', z.boolean()),
```